pkg/exchange/binance/exchange.go

```diff
@@ -448,6 +489,16 @@ func (e *Exchange) QueryOpenOrders(ctx context.Context, symbol string) (orders [
 		}
 
 		return toGlobalOrders(binanceOrders)
+	} else if e.IsFutures {
+		req := e.futuresClient.NewListOpenOrdersService().Symbol(symbol)
+		// req.IsIsolated(e.IsIsolatedFutures)
```

Comment:
did not set this?

Reply:
I see. It's redundant.
Remove code block after else if from line 524,  too?

But, How about the use of toGlobalFuturesOrders function from convert.go?

Reply:
Done (Removed). both of them.